courses/scratch/ghostcatcher.yml

```
@@ -0,0 +1,310 @@
+
+      First select "No outline" so that ther's no black border around the shapes
+
+      ![](assets/no_outline.png)
+
+      Let's draw a circle and a square on top of it.
+      
+      ![](assets/circle_and_square_for_ghost.png)
+
+      You can draw the ghost's legs by erasing two triangles from the bottom
+
+      ![](assets/ghost_legs.png)
+
+      Select "white" as fill colour (bring Saturation to 0) and draw an "oval" shape to make the eye.
+
+      ![](assets/ghost_eye.png)
+
+      *Trick*: to draw the second eye you can press `CTRL-C` and `CTRL-V` to "copy-paste" the previous one.
+
+      Let's give the eyes a final touch
+
+      ![](assets/ghost_eyes.png)
+
+  - subtitle: (Optional) Animating the ghost
+    content: |
+      Feel free to come to this section later if you want to go straight to continuing with the game.
+
+      Let's make an animated ghost just like we did for pacman.
+      
+      Duplicate the costume ghost.
+
+      Click on the black ovan inside the eye.
+
+      *While holding down the _shift_ key* click on the black oval inside the other eye.
+
+      Move the eyes together by pressing the left arrow.
+
+      ![](assets/ghost_animated_eyes.png)
+
+      Now let's add to the ghost the same animation code we used for Pacman
+
+      ![](assets/pacman_animation_cycle.png)
+
+  - subtitle: Let's make the ghost move
+    content: |
+      Click on the "Code" tab of the ghost sprite, and add the following blocks:
+
+      ![](assets/ghost_motion.png)
+
+      You'll notice the ghost turns into all directions. In order to make it "stand", add the "set rotation style to left-right":
+
+      ![](assets/ghost_motion_2.png)
+
+      If the ghost bounces in a "weird" way, go back to the costumes and make sure the ghost is positioned in the center of the drawing area.
+
+  - subtitle: Pacman eats ghost!
+    content: |
+      Let's import a sound that Pacman will make when "eating" a ghost.
+
+      Click on the "ghost" sprite
+
+      ![](assets/select_ghost_sprite.png)
+
+      then click on the "sounds" tab and on the "select sound" at the bottom
+
+      ![](assets/choose_sound.png)
+
+      for example let's use the "chomp" sound:
+
+      ![](assets/select_chomp_sound.png)
+
+      Let's rename the sprites in order to make our life easier in a moment. Click on each sprite and change the name in the "Sprite" section:
+
+      ![](assets/rename_sprites.png)
+
+      Let's make Pacman "eat" a ghost! Modify the existing code so that it looks like this:
+
+      ![](assets/pacman_eats_ghost.png)
+
+      When a ghost is touching pacman, we make it disappear for 1 second and reappear in a random position.
+
+  - subtitle: Adding more ghosts
+    content: |
+      First of all let's make pacman and the ghost smaller, in order to make room for more ghosts
+
+      Let's use the block "Change size to <percentage>". for example 60%.
```

```suggestion
      Let's use the block `Change size to <percentage>`. Try a value of `60%`.
```

You need some way of escaping the `<percentage>` otherwise it treats it as a tag. You can either use a code block (like my example) or you can use the `&lt;` and `&gt;` codes.